tests/internals/global_custom_ca/global_custom_ca_test.go

```diff
@@ -0,0 +1,284 @@
+//go:build e2e
+// +build e2e
+
+package global_custom_ca_test
```

Comment:
I wonder whether this test should be in the `internals` package? Shouldn't we move it out and put it somewhere to near `chaos` together with the changes in the `setup_test.go`. To not modify default KEDA installation with custom cersts for other tests, WDYT?

Reply:
I think that this doesn't affect other tests because it's a configuration thing so I'd not move it but if you think that excluding it is better, I'd create another issue to unify all the "other" test cases under "sequential" or "single" executed test. Just splitting this is a mess because the execution script needs to be modified to trigger the new folder

Reply:
Agree, @JorTurFer please create the issue. I think that we should have the `sequential` tests.

Reply:
https://github.com/kedacore/keda/issues/4244